<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>656</width>
    <height>311</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>850</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Install Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../png/maneki_neko.png</normaloff>../png/maneki_neko.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(30, 31, 47);
	
	color: rgb(118, 159, 240);
};

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>124</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>114</width>
             <height>124</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="manekiIcon">
              <property name="minimumSize">
               <size>
                <width>98</width>
                <height>96</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>98</width>
                <height>96</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../png/maneki_neko.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="headingLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 57 14pt &quot;JetBrains Mono&quot;;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You're abt to install N programs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="Line" name="line">
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="commentLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>23</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 57 10pt &quot;JetBrains Mono&quot;;</string>
              </property>
              <property name="text">
               <string>The programs you selected are:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="installQueueLabel">
              <property name="minimumSize">
               <size>
                <width>486</width>
                <height>93</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>Installation will proceed on a new terminal window.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="proceedButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Proceed</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
